apps/dc_tools/tests/test_stac_transform.py

```diff
@@ -128,6 +128,11 @@ def test_usgs_landsat_stac_transform(usgs_landsat_stac) -> None:
     )
 
 
+def test_stac_1_1(stac_1_1) -> None:
+    transformed_stac_doc = stac_transform(stac_1_1)
+    assert transformed_stac_doc["crs"] == "epsg:32656"
```

Comment:
I don't know anything about the specification, but the examples I've seen use capital letters for `EPSG` (search for `EPSG:3857` in https://cloudnativegeo.org/blog/2024/09/stac-1.1.0-released/ to see one example).

Reply:
Proj (and therefore and therefore pretty much everyone) is case insensitive and lower-case has historically been common in the odc world for unclear reasons.  This is testing for an internal representation for odc use, so it's fine.

Reply:
Given the code and epsg are different things, may as well bite the bullet on this and be consistent.